tasks/Mock Jury.md

```diff
@@ -0,0 +1,42 @@
+# Summary
+The experiment performed was designed to test the effects of group deliberation on jurors when exposed to inadmissable evidence.
+Speciffically the study sought to see how group deliberation would effect jurors individual ability to disregard inadmissable evidence.
+
+# References
+London, K., & Nunez, N. (2000). The effect of jury deliberations on jurors' propensity to disregard inadmissible evidence. Journal of Applied Psychology, 85(6), 932–939. https://doi.org/10.1037/0021-9010.85.6.932
+
+Link: https://psycnet-apa-org.proxy.library.upenn.edu/record/2000-16508-010
+
+# Stimuli
+## The visual components
+N/A 
+
+## Materials for alternative versions of the experiment 
+3 versions of a mock trial script.  All versions of the script should contain testimones from both the defense and the prosecution.  The only difference in the versions of the scipts are the extra admissable/inadmissable evidence. The three versions should be as follows:
```

Comment:
I don't think these would be alternative experiments but different conditions within the same experiment- I would maybe move this into the procedure section to describe the differences between the different treatment groups.